docs/en/Configuring-Alluxio-with-S3.md

```diff
@@ -91,6 +91,16 @@ The `jets3t` version `0.9.0` works for Hadoop version `2.3.0`. The `jets3t` vers
 work for older versions of Hadoop. To find the exact `jets3t` version for your Hadoop version,
 please refer to [MvnRepository](http://mvnrepository.com/).
 
+## Use non-amazon service provider
+
+If you have a service provider other than "s3.amazonaws.com", modify the `ALLUXIO_JAVA_OPTS` section of `conf/alluxio-env.sh` to include:
+
+{% include Configuring-Alluxio-with-S3/non-amazon.md %}
+
+For these parameters, replace `<S3_ENDPOINT>` with the host name of your S3 service. Set `<USE_HTTPS>` to `true` or `false`, then set the appropriate port number; without those values the system will default to 80 for HTTP and 443 for HTTPS. You can remove the unused port entry. 
+
+The underlying S3 library JetS3t can incorporate bucket names that are DNS-compatible into the host name of its requests when `<DISABLE_DNS>` is set to `false`. For example, a request directed at the bucket named "mybucket" will be sent to the host name "mybucket.s3.amazonaws.com". If you set `<DISABLE_DNS>` to `true`, JetS3t will specify bucket names in the request path of the HTTP message instead of the Host header, for example: "http://s3.amazonaws.com/mybucket". Without this parameter set, the system will default to `false`. See http://www.jets3t.org/toolkit/configuration.html for further details.
```

Comment:
For this paragraph (line 102), does it only apply to "non-amazon service provider", or it is true in general (in this case, probably put it as a separate section?)